src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine{

```

Comment:
Please use a space between the superclass `Machine` and the curly brace `{` to follow Java code style conventions.